xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3077,6 +3077,8 @@ platform.livetable.paginationResultsSingle=Result <span class="currentResultsNo"
 platform.livetable.paginationResultsMany=Results <span class="currentResultsNo">{0} - {1}</span> of <span class="totalResultsNo">{2}</span>
 platform.livetable.paginationResults=Results
 platform.livetable.paginationResultsOf=out of
+platform.livetable.false=False
+platform.livetable.true=True
```

Comment:
yes of course (old PR)
Anyway, now I found the way to identify property type (boolean) but didn't found time to add it to avoid new 'listboolean' option...